src/material-experimental/mdc-list/list.ts

```diff
@@ -29,9 +29,9 @@ import {MatListBase, MatListItemBase} from './list-base';
 @Component({
   selector: 'mat-list',
   exportAs: 'matList',
-  template: '<ng-content></ng-content>',
+  template: '<div class="mdc-deprecated-list"><ng-content></ng-content></div>',
   host: {
-    'class': 'mat-mdc-list mat-mdc-list-base mdc-deprecated-list',
+    'class': 'mat-mdc-list mat-mdc-list-base',
```

Comment:
Is there a better way we can avoid style conflicts? I think this makes it a bit inconvenient in terms of instantiating the `MDCList` foundation, and declaring the adapter. I'm not sure if this currently has any a11y impact.

https://github.com/angular/components/blob/ddbfe11e9698f5b252f6eb9d9997a1f88505faa2/src/material-experimental/mdc-list/interactive-list-base.ts#L160

Reply:
Agreed that this isn't great, but the alternative would be to fix it in the select, autocomplete and menu specifically, but then it could still conflict with other list-based components in the future or other non-Material components the user might have on the page.

Reply:
We should probably change the adapter to apply any classes, attributes, etc to the div. The only other way I can think to do this is if MDC changed their mixins to allow us to add to the root selector:

```scss
@mixin deprecated-without-ripple(..., $root: '') {
  #{$root}.mdc-deprecated-list {
    ...
  }
}
```

Reply:
@mmalerba Yeah, I thought that too, but then realized that this could break a11y because the `MDCList` "root" usually receives the `aria-` attributes from the foundation.

Personally I'd prefer what you showed. We _might_ be able to convince MDC to support this because list is commonly used in various other components too, and it makes style isolation easier.